Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +10,315 @@ public static class BalanceStats
+                    moreThanOne = false;
+                    result = $" is the richest person. {max:C0}.";
+                }
+                else if(max.Equals(currentBalance))
+                {
+                    richestPeople += $", {auxBalances[0]}";
+                    result = $" are the richest people. {max:C0}.";
+                    moreThanOne = true;
+                }
+            }
+
+            if (moreThanOne)
+            {
+                richestPeople = ReplaceLastCommaWithAnd(richestPeople);
+            }
+
+            return richestPeople + result;
         }
 
         /// <summary>
         /// Return name and current money of the most poor person.
         /// </summary>
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (IsStrArrayNullOrEmpty(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+
+            CultureInfo.CurrentCulture = GetCustomInvariantCulture();
+
+            decimal min = decimal.MaxValue;
+            string result = "";
+            string poorPeople = "";
+            bool moreThanOne = false;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
+            {
+                string[] auxBalances = peopleAndBalances[i].Split(',');
+                string value = auxBalances[^1..][0];
+                decimal.TryParse(value, out decimal currentBalance);
+                if (min > currentBalance)
+                {
+                    min = currentBalance;
+                    poorPeople = auxBalances[0];
+                    result = $" has the least money. {min:C0}.";
+                    moreThanOne = false;
+                }
+                else if(min.Equals(currentBalance))
+                {
+                    poorPeople += $", {auxBalances[0]}";
+                    result = $" have the least money. {min:C0}.";
+                    moreThanOne = true;
+                }
+            }
+
+            if (moreThanOne)
+            {
+                poorPeople = ReplaceLastCommaWithAnd(poorPeople);
+            }
+
+            return poorPeople + result;
+        }
+
+        private static CultureInfo GetCustomInvariantCulture()
+        {
+            CultureInfo cult = (CultureInfo)CultureInfo.InvariantCulture.Clone();
+            NumberFormatInfo nfi = cult.NumberFormat;
+            nfi.CurrencyNegativePattern = 1;
+            nfi.CurrencyGroupSeparator = "";
+
+            return cult;
+        }
+
+        private static bool IsStrArrayNullOrEmpty(string[] array) 
+        {
+            if (array == null || array.Length == 0)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        private static bool IsDecArrayNullOrEmpty(decimal[] array)
+        {
+            if (array == null || array.Length == 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '?:' operator here.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5069037)